files/usr/local/etc/uwsgi/uwsgi.ini

```diff
@@ -0,0 +1,39 @@
+[uwsgi]
+chdir = /opt/taiga-back
+die-on-term = true
+exit-on-reload = true
+uid = uwsgi
+gid = uwsgi
+master = true
+module = taiga.wsgi
+static-expires-uri = ^/(static|v-\d+)/ 31536000
+static-safe = /etc/opt/taiga-front/conf.json
+static-safe = /opt/taiga-front
+static-safe = /var/opt/taiga-back/media
```

Comment:
Why is it not needed to specify `/srv/taiga-back` as `static-safe`, if route-label `static` serves files from there?

Reply:
Don't know, just works.

Reply:
Hm, it would appear that the file has to be it path specified by `static:` route OR in one of `static-safe` paths. Which also makes `static-safe = /var/opt/taiga-back/media` redundant if `route-label` media below is rewritten properly.

Reply:
In 2958fbe63c7247be7ee85b563f5abd3fdf10fde8, only `/etc/opt/taiga-front/conf.json` is explicitly marked as safe.